controller/src/main/java/io/pravega/controller/server/SegmentHelper.java

```diff
@@ -546,13 +546,22 @@ public CompletableFuture<Void> removeTableKeys(final String tableName,
                                                    final List<TableSegmentKey> keys,
                                                    String delegationToken,
                                                    final long clientRequestId) {
+
+        return removeTableKeys(tableName, ModelHelper.encode(uri), keys, delegationToken, clientRequestId);
+    }
+
+    public CompletableFuture<Void> removeTableKeys(final String tableName,
+                                                   PravegaNodeUri uri,
+                                                   final List<TableSegmentKey> keys,
+                                                   String delegationToken,
+                                                   final long clientRequestId) {
```

Comment:
Why we need to change these methods, can't we use the existing one?

Reply:
Updated to use the existing method.